include/graphblas/reference/blas1.hpp

```diff
@@ -9759,7 +9760,10 @@ namespace grb {
 	 * @see Vector::operator[]()
 	 * @see Vector::lambda_reference
 	 */
-	template< typename Func, typename DataType, typename Coords >
+	template<
+		Descriptor descr = descriptors::no_operation,
```

Comment:
The function should return illegal if x was not dense but the dense descriptor was passed (maybe this check is also missing with the other backends)